xbmc/Application.cpp

```diff
@@ -2829,7 +2829,7 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
         options.state = item.GetProperty("savedplayerstate").asString();
         item.ClearProperty("savedplayerstate");
       }
-      else if (item.m_lStartOffset == STARTOFFSET_RESUME)
+      else if ((item.m_lStartOffset == STARTOFFSET_RESUME) && !(item.HasProperty("StartPercent") && options.startpercent == 0.0f))
```

Comment:
`item.m_lStartOffset == STARTOFFSET_RESUME` does not need parenthesis

Reply:
Will correct that, thank you.